<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteInfoByWavDialog</class>
 <widget class="QDialog" name="DeleteInfoByWavDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>250</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>删除数据</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择一个之前导入的音频文件，删除与此文件关联的数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>音频文件：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_lable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_files">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
